components/tools/OmeroPy/src/omero/util/cleanse.py

```diff
@@ -189,7 +189,17 @@ def __str__(self):
             (len(self.cleansed), self.bytes_cleansed)
 
 
-def initial_check(config_service):
+def initial_check(admin_service, config_service):
```

Comment:
I don't think this code is used elsewhere, but the API-safer pythonic way of doing this would be:

```
def initial_check(config_service, admin_service=None):
```

though I'd err on the side of:

```
def initial_check(config_service, admin_service):
    if admin_service is None: raise...
```